server/repositories/offender.js

```diff
@@ -5,61 +5,72 @@ function validateOffenderNumberFor(offenderNo) {
   return pattern.test(offenderNo);
 }
 
-const baseUrl = `${config.prisonApi.baseUrl}/api`;
+const prisonApiBaseUrl = `${config.prisonApi.baseUrl}/api`;
+const incentivesApiBaseUrl = `${config.incentivesApi.baseUrl}`;
 
-function offenderRepository(httpClient) {
+function offenderRepository(prisonApiHttpClient, incentivesApiHttpClient) {
   function getOffenderDetailsFor(offenderNo) {
     if (validateOffenderNumberFor(offenderNo)) {
-      return httpClient.get(
-        `${baseUrl}/bookings/offenderNo/${offenderNo.toUpperCase()}`,
+      return prisonApiHttpClient.get(
+        `${prisonApiBaseUrl}/bookings/offenderNo/${offenderNo.toUpperCase()}`,
       );
     }
     throw new Error('Invalid offender number');
   }
 
   function getIncentivesSummaryFor(bookingId) {
-    return httpClient.get(`${baseUrl}/bookings/${bookingId}/iepSummary`);
+    return incentivesApiHttpClient.get(
+      `${incentivesApiBaseUrl}/iep/reviews/booking/${bookingId}/iepSummary`,
```

Comment:
This URL api entry looks odd - iepSummary at the end?

Reply:
Good spot! I was mixing up Prison API and Incentives API endpoints